src/client/pages/LoginApp/LoginAppScreen.tsx

```diff
@@ -59,7 +79,11 @@ export const LoginAppScreen: React.FC = () => {
         <title>{textKeys.LOGIN_APP_PAGE_TITLE()}</title>
       </Helmet>
       <Page>
-        <TopBar centered />
+        <Header>
+          <LogoLink href={'/' + pathLocale}>
+            <HedvigLogo width={94} />
+          </LogoLink>
```

Comment:
Since this is only used as a web view in the apps (actually from what it looked like on my phone, not on iOS but actually just in the Android app) I think the link wrapping the logo is pretty redundant and can be removed.

Reply:
I removed the pathLocale and changed the name to Logo so we can still have a parent div to center the logo.